src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/UniversalRemoteAuthenticationProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages
+{
+    /// <summary>
+    /// Implementation of IRemoteAuthenticationProvider for the "Universal" authentication type.
+    /// </summary>
+    [Export(typeof(IRemoteAuthenticationProvider))]
+    [AppliesTo("WinUI")] // TODO: ???
+    internal class UniversalRemoteAuthenticationProvider : IRemoteAuthenticationProvider
```

Comment:
This can be removed before merge, its speculative, but illustrates why IRemoteAuthenticationProvider exists

Reply:
What is it about "universal" that causes us to supplier this port supplier? What does a port supplier actually provide? Is it go to do with the fact that these apps are in an AppContainer? 

(Trying to understand when/when not we would turn this on)

Reply:
Oh this is an authentication mode?

Reply:
Yes, this is providing "Universal Auth". It's used by UWP and Desktop Bridge.

The "when" question in my mind would be answered by whatever decides to add the capability, and I was 50/50 on whether the capability for this should be "WinUI" or "SupportsUniversalAuth", but in light of recent news I stopped thinking about it and instead I figure I'll remove it before I merge anyway so it doesn't matter :)

Reply:
I updated the capability on this